### PR TITLE
Template updates

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

Since #7183 rendered the two keyboard.c templates basically identical. Didn't have to modify the script so I assume the Python version (if it's landed already) doesn't need any changes either.

While I'm here, should I also turn the trailing double spaces section in the readme into a list? Or use `<br>`s? Or?
And perhaps the info.json could have something in it, too. @yanfali?

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
> While I'm here, should I also turn the trailing double spaces section in the readme into a list? Or use `<br>`s? Or?

Please, *please*, **PLEASE**, I beg of you – turn this into a list.


> And perhaps the info.json could have something in it, too.

My suggestion for this would be something like:

```json
{
    "keyboard_name": "%KEYBOARD%",
    "url": "",
    "maintainer": "%YOUR_NAME%",
    "width": 3,
    "height": 2,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"label":"k00", "x":0, "y":0},
                {"label":"k01", "x":1, "y":0},
                {"label":"k02", "x":2, "y":0},
                {"label":"k10", "x":0, "y":1, "w":1.5},
                {"label":"k12", "x":1.5, "y":1, "w":1.5}
            ]
        }
    }
}
```
(human-readable and conforms to the example [given in the docs](https://docs.qmk.fm/#/hand_wire?id=getting-some-basic-firmware-set-up))


And could we change https://github.com/qmk/qmk_firmware/blob/742e83e102f5d91087672ae3f91e433a74bd51dd/quantum/template/base/keyboard.h#L28-L35 to:

```c
#define LAYOUT( \
    k00, k01, k02, \
      k10,  k12   \
) \
{ \
    { k00, k01,   k02 }, \
    { k10, KC_NO, k12 }, \
}
```

so this block conforms to `k<row><column>` notation?

**Comment by fauxpark** [comment]
Done, and I also added some stuff to the keymap.c template to demonstrate simple layering.

File: quantum/template/base/keymaps/default/keymap.c [hunk 1]
```diff
@@ -15,17 +15,28 @@
  */
 #include QMK_KEYBOARD_H
 
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+}
```
Comment (noroadsleft) [inline quantum/template/base/keymaps/default/keymap.c]: I don't know if it's just me, but my keymaps never compile without a semicolon here.

```suggestion
};
```

Comment (fauxpark) [reply quantum/template/base/keymaps/default/keymap.c]: 🤦‍♂

**Pull request merged**